frontend/.env

```diff
@@ -1,5 +0,0 @@
-REACT_APP_API_URL=/api
-
-BROWSER=none
-PORT=3000
-PUBLIC_URL=/
```

Comment:
Why did you remove this part? Have you discussed these changes with someone?

Reply:
oh yes( I tested SPA and forgot return this settings.I will correct its immediately.